test/app.cpp

```diff
@@ -6,6 +6,7 @@
 #include "UFDecoder.hpp"
 #include "Utils.hpp"
 
+#include <cmath> /* pow */
```

Comment:
The changes throughout this file do not really seem related to this PR. Although, I am fine with them, as they are quite minor. So if these changes are intentional, then keep them here.